styles/globals.scss

```diff
@@ -1,6 +1,7 @@
 @import './colors';
 @import '../components/common/navbar/navbar.scss';
 @import '../components/resume/resume.scss';
+@import '../pages/index.scss';
```

Comment:
As I add more and more sections/components with styles, adding this import manually is getting a bit tedious. Make a task to look into setting up scss glob imports.